### PR TITLE
Added changes I've made to README.md

### PR DESCRIPTION
**Pull request by vocatwk**
I've added changes I can think of to our README.md. This isn't really worthy of a pull request but I didn't want to make a change to README directly just incase you guys wanted to change things or this isn't what you had in mind.

### CODE REVIEW

**Comment by lealgamb** [comment]
Anything is worthy of a PR 👍 Will add some of my stuff after this is merged to avoid conflicts.

**Comment by yebrahim** [comment]
I'd say, if you've gone through the trouble of creating a PR for it, I'll go through the trouble of code reviewing it. What "it" is doesn't matter at all.

.. that's of course so long as you don't have an automated bot sending one-line PRs at a time! 😛

File: README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
 # CodeU Example Project
```
Comment (yebrahim) [inline README.md]: Change this title as well?

Comment (vocatwk) [reply README.md]: Will do.

**Pull request merged**